uctMaths/competition/reports.py

```diff
@@ -54,13 +55,32 @@ def send_answer_sheets(school, answer_sheet, cc_admin=False):
                     'The UCT Mathematics Competition team'%(rteacher.firstname + " " + rteacher.surname, school.name)
     output_string += UMC_header("Answer Sheets")
     output_string += 'Answer sheets for %s\n%s\n'%(school.name, UMC_datetime())
-    recipient_list = [rteacher.email]
+    recipient_list = [rteacher.email_school]
     if cc_admin:
         recipient_list.append(compadmin.admin_emailaddress())
+    send_email(
+        "(Do not reply) UCT Mathematics Competition %s Answer Sheets %s" % (school.name, recipient_list),
+        output_string,
+        "UCT Mathematics Competition <UCTMathsCompetition@j5int.com>",
```

Comment:
This address doesn't exist and j5int.com isn't a domain that we use any more. I think this should change to something like noreply@uctmathscompetition.org.za?
There are other places in the site that also use j5int.com addresses. They should be updated as well.

Reply:
I don't think we should use a fake email address, as spam filters might not like that. What is the address are we sending emails from?

Reply:
So I spoke to Stephen, and it sounds like our old mail server has been taken down and we need to reconfigure this.

I think we should leave this as is for this MR and fix the issues with the mail server in a different MR.